GetterPriceBinance.py

```diff
@@ -0,0 +1,70 @@
+import asyncio
+import logging
+
+from binance.lib.utils import config_logging
+from binance.websocket.spot.websocket_client import (
+    SpotWebsocketClient as WebsocketClient,
+)
+
+config_logging(logging, logging.DEBUG)
+
+
+class book_ticker_price_binance:
+    def __init__(self, symbols=None, symbol=None):
```

Comment:
We can, but we will get only what's called "combined streams". They looks similar to this:
`        {'stream': 'srmbtc@depth20@100ms', 'data': {'lastUpdateId': 596673039, 
         'bids': [['0.00004980', '16266.80000000'], ['0.00004970', '32377.40000000'], ['0.00004960', '20167.30000000'],
                     ['0.00004950', '4060.60000000'],  ['0.00004940', '10222.00000000'], ['0.00004930', '2300.10000000'], 
                     ['0.00004920', '2231.50000000'], ['0.00004910', '3.30000000'], ['0.00004900', '7.70000000'],
                     ['0.00004890', '2060.20000000'], ['0.00004880', '2015.10000000'], ['0.00004870', '6.00000000'], 
                     ['0.00004860', '35.70000000'],  ['0.00004850', '150.40000000'], ['0.00004840', '2.10000000'],
                     ['0.00004830', '2.30000000'],   ['0.00004820', '215.00000000'], ['0.00004810', '2.30000000'],
                     ['0.00004800', '217.70000000'],   ['0.00004790', '68.70000000']], 
         'asks': [['0.00005000', '2365.50000000'], ['0.00005010', '7460.30000000'], ['0.00005020', '41115.80000000'], 
                     ['0.00005030', '13960.40000000'], ['0.00005040', '4899.00000000'], ['0.00005050', '7765.70000000'], 
                     ['0.00005060', '850.40000000'], ['0.00005080', '1083.30000000'], ['0.00005090', '2.30000000'],
                     ['0.00005100', '488.00000000'], ['0.00005140', '891.20000000'], ['0.00005150', '6.00000000'], 
                     ['0.00005160', '39.10000000'], ['0.00005210', '2.20000000'], ['0.00005220', '86.30000000'], 
                     ['0.00005230', '20.60000000'], ['0.00005250', '9.50000000'], ['0.00005260', '65.90000000'],
                     ['0.00005270', '146.70000000'], ['0.00005280', '14.80000000']]}}`
Though, if you have, let's say, one or a few streams, binance-connector will use what's called "raw streams". They have next issue: if you have more than 5-10 streams, you'll get info of one stream to another stream. This caused because there's no way to improve internet latency. They're similar to this:
 `{'lastUpdateId': 596673039, 
         'bids': [['0.00004980', '16266.80000000'], ['0.00004970', '32377.40000000'], ['0.00004960', '20167.30000000'],
                     ['0.00004950', '4060.60000000'],  ['0.00004940', '10222.00000000'], ['0.00004930', '2300.10000000'], 
                     ['0.00004920', '2231.50000000'], ['0.00004910', '3.30000000'], ['0.00004900', '7.70000000'],
                     ['0.00004890', '2060.20000000'], ['0.00004880', '2015.10000000'], ['0.00004870', '6.00000000'], 
                     ['0.00004860', '35.70000000'],  ['0.00004850', '150.40000000'], ['0.00004840', '2.10000000'],
                     ['0.00004830', '2.30000000'],   ['0.00004820', '215.00000000'], ['0.00004810', '2.30000000'],
                     ['0.00004800', '217.70000000'],   ['0.00004790', '68.70000000']], 
         'asks': [['0.00005000', '2365.50000000'], ['0.00005010', '7460.30000000'], ['0.00005020', '41115.80000000'], 
                     ['0.00005030', '13960.40000000'], ['0.00005040', '4899.00000000'], ['0.00005050', '7765.70000000'], 
                     ['0.00005060', '850.40000000'], ['0.00005080', '1083.30000000'], ['0.00005090', '2.30000000'],
                     ['0.00005100', '488.00000000'], ['0.00005140', '891.20000000'], ['0.00005150', '6.00000000'], 
                     ['0.00005160', '39.10000000'], ['0.00005210', '2.20000000'], ['0.00005220', '86.30000000'], 
                     ['0.00005230', '20.60000000'], ['0.00005250', '9.50000000'], ['0.00005260', '65.90000000'],
                     ['0.00005270', '146.70000000'], ['0.00005280', '14.80000000']]}`
So, we can, but we lose in amount of needed network connection.